ETL: Improve tutorial about InfluxDB (#89)
crate/cratedb-guide: docs/integrate/etl/influxdb.md

(import-influxdb)=
# Import data from InfluxDB

> In this quick tutorial, you will use the InfluxDB I/O subsystem of CrateDB Toolkit

Reviewer: ```suggestion
In this quick tutorial, you'll use the CrateDB Toolkit's InfluxDB I/O subsystem.
```

Minor phrasing suggestion.
